### DISCUSSION TITLE
When do you expect jit to become stable?

### DISCUSSION THREAD

**Discussion by Chetan11-dev** [tailwindlabs/tailwindcss#5345] [General]
When does the tailwind team expect jit to become stable?

**Comment by lukewarlow** [comment]
JIT is the default (and only mode) available in the upcoming Tailwind 3.0 release. I've no clue on ETAs (I'm just a rando who's looked at the commit logs)

**Reply by simonswiss** [reply]
Correct, we're currently working towards making JIT the _default_ mode in Tailwind CSS v3.0.

The release is definitely a few months away from now, but that should give you confidence to start using `JIT` in projects. It's already pretty stable and used in production sites as of now 👍
